File: 0268-missing-number/0268-missing-number.cpp
class Solution {
    public:
int missingNumber(vector<int>& nums) {
     int total, sum=0, m, n;
     n= nums.size();
    total= n*(n+1)/2;
    for (int i=0; i<n; ++i){
        sum += nums[i];
    }
return (total -sum);
}
};

File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n= len(nums)
        total_sum = (n*(n+1))//2

        arr_sum= sum(nums)

        return(total_sum-arr_sum) # this is the easiest method but now i will try to do with the bit wise operator;